portal/models/fhir.py

```diff
@@ -660,7 +661,7 @@ def fetch_HL7_V3_Namespace(valueSet):
 def fetch_local_valueset(valueSet):
     """Pull and parse the named valueSet from our local definition"""
     response = valueset_nhhd_291036()
-    data = json.loads(response.data)
+    data = json.loads(response.data.decode("utf-8"))
```

Comment:
@pbugni Is there a reason we can't use `response.json` to get the JSON directly?

Reply:
excellent point, @ivan-c 

@John0Randolph you can kill off the `data` variable, and use `response.json()['codeSystem']['concept']` in the call to `parse_concepts`  - note a similar patch should be applied to `fetch_HL7_V3_Namespace()` just above.

Reply:
This fix works in fetch_HL7_V3_Namespace(), but when I put it in fetch_local_valueset(valueSet), it throws an error (although I might be implementing it wrong). Code:
    response = valueset_nhhd_291036()
    #data = json.loads(response.data.decode("utf-8"))
    return parse_concepts(response.json()['codeSystem']['concept'],
                          system='{}/{}'.format(TRUENTH_VALUESET, valueSet))

Error:
TypeError: 'dict' object is not callable

Reply:
json is a property, not a method :)

`return parse_concepts(response.json`~~()~~`['codeSystem']['concept']`